src/math/trig.c

```diff
@@ -100,3 +166,5 @@ void cordic_sin_cos(int32_t th_rad_fxp, cordic_cfg type, int32_t *sign, int32_t
 }
 
 #endif
+
+#endif
```

Comment:
add comments here so its clear what `if` this is closing please

Reply:
Yep, done!

Reply:
is it pushed?